src/main/java/meteordevelopment/meteorclient/systems/modules/combat/AnchorAura.java

```diff
@@ -313,7 +314,7 @@ private BlockPos findPlacePos(BlockPos targetPlacePos) {
                 else if (isValidPlace(targetPlacePos.add(1, 1, 0))) return targetPlacePos.add(1, 1, 0);
                 else if (isValidPlace(targetPlacePos.add(-1, -1, 0))) return targetPlacePos.add(-1, -1, 0);
                 else if (isValidPlace(targetPlacePos.add(0, 1, 1))) return targetPlacePos.add(0, 1, 1);
-                else if (isValidPlace(targetPlacePos.add(0, 0, -1))) return targetPlacePos.add(0, 0, -1);
+                else if (isValidPlace(targetPlacePos.add(0, 0, -1))) return targetPlacePos.add(0, 0, -1); //Duplicate of line 313
```

Comment:
As for the duplicate lines, why don't you just remove them yourself? You've commented on them anyways, imo its better to just delete it instead of waiting for someone else.

Reply:
because i feel like those duplicate lines were supposed to actually point to a unique block and it is currently being ignored
so im waiting on someone who actually knows how this module works to fix it

Reply:
Since this all happens within a single tick, they won't be pointing to a different block. Probably just a forgotten left over line from duplications during writing.